src/spam-protection/get-google-recaptcha-token.ts

```diff
@@ -0,0 +1,15 @@
+import { ReadableDataStore } from '@bigcommerce/data-store';
+
+import { InternalCheckoutSelectors } from '../checkout';
+
+import CardingProtectionActionCreator from './carding-protection-action-creator';
+
+export default function getGoogleRecapthaToken(store: ReadableDataStore<InternalCheckoutSelectors>, error: any) {
+    const { additional_action_required, status } = error;
+    if (status === 'additional_action_required' && additional_action_required && additional_action_required.type === 'recaptcha_v2_verification') {
```

Comment:
Same condition as: https://github.com/bigcommerce/checkout-sdk-js/pull/875/files#diff-6403927dbab7260b0f1154bc8ce53ee6R37 ?

This again makes me think it might be nice to think the error body could be a union type and this conditional could be a type guard. Possibly used at the higher scope:

```
isRecaptchaRequiredError(error)
```